xml/obs_scm_ci_workflow_integration.xml

```diff
@@ -0,0 +1,1149 @@
+          &gt; Edit &gt; Regenerate Token</emphasis>.</para>
+
+          <para><emphasis role="bold">b) Delete the token</emphasis></para>
+
+          <para>You can always delete your token via WebUI, in
+          <emphasis>Profile &gt; Manage Your Tokens</emphasis>, or with these
+          commands:</para>
+
+          <screen language="xml"><command>osc</command> token # list all your tokens</screen>
+
+          <screen language="xml"><command>osc</command> token --delete $token_id # remove the token with the given id</screen>
+
+          <para>Then you can create a new one as explained in <xref
+          linkend="sec.obs.obs_scm_ci_workflow_integration.setup.token_authentication.how_to_authenticate_scm_with_obs"/>
+          and replace it wherever you use it.</para>
+        </sect4>
+      </sect3>
+    </sect2>
+
+    <sect2 xml:id="sec.obs.obs_scm_ci_workflow_integration.setup.webhooks">
+      <title>Webhooks</title>
+
+      <para>Once OBS and GitHub are allowed to speak to each other, they can
+      start talking via <emphasis role="bold">webhooks</emphasis>.</para>
+
+      <sect3 xml:id="sec.obs.obs_scm_ci_workflow_integration.setup.webhooks.scm_events">
+        <title>SCM Events</title>
+
+        <para>On a GitHub repository, events are happening all the time: a
+        pull request is created, somebody pushes a commit, a pull request is
+        merged etc. When you set up a webhook on GitHub, you can specify
+        which events you are interested in. Only when those events happen, the
+        webhook will be sent to OBS.</para>
+
+        <para>This is the list of SCM events supported by the existing
+        workflows in OBS:</para>
+
+        <itemizedlist>
+          <listitem>
+            <para>Pull requests</para>
+          </listitem>
+
+          <listitem>
+            <para>Pushes</para>
+          </listitem>
+        </itemizedlist>
+
+        <para>Other SCMs like GitLab have similar behaviour; the main
+        difference is how they name the events. Refer to the <link
+        linkend="sec.obs.obs_scm_ci_workflow_integration.setup.equivalence_table">Equivalence
+        Table</link> for more details or read more about <link
+        xlink:href="https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads">GitHub
+        events</link> and <link
+        xlink:href="https://docs.gitlab.com/ce/user/project/integrations/webhooks.html#events">GitLab
+        events</link>.</para>
+      </sect3>
+
+      <sect3 xml:id="sec.obs.obs_scm_ci_workflow_integration.setup.webhooks.how_to_set_up_a_webhook_on_github">
+        <title>How to Set Up a Webhook on Github</title>
+
+        <para>Go to the project you want to set the integration on, then under
+        <emphasis>Settings &gt; Webhooks</emphasis>.</para>
+
+        <figure>
+          <title>Webhooks on GitHub.</title>
+
+          <mediaobject>
+            <imageobject>
+              <imagedata contentwidth="931"
```

Comment:
Not sure if I generated the pdf wrong, but for me the screenshot does not fit on the page (to wide)

Reply:
Having another look...